README.md

```diff
@@ -1 +1,55 @@
+
+## Questers
+
+Questers are users of the application who wish to compete to complete  quests and earn BANANO tokens. Questers must have a nominal amount of Ether in their account to complete them. A quester chooses a quest they want to complete, read the hint associated with it, and msut travel to a physical area in the world to where they believe the quest can be completed. If correct when they submit the quest, an augmented reality banano shows up that they can take a screenshot and share with their friends.  
+
+# Tech Stack
+
+The Pocket team built a custom set of extensible tools to make it easier for developers who wish to build peer to peer applications. Pocket is blockchain agnostic, and everything is built with the goal of easily adding a plugin for any other blockchain into the ecosystem. By doing so, Pocket can be a simple, common interface for developers who want to use a specific cryptocurrency or multiple ones in their applications.  
+
+## Ethereum
+
+We chose Ethereum as the decentralized "backend" for BANANO Quest due to our familiarity and maturity of the tools and network. While confirmation times and lack of avoiding users having to use Ether is a significant UX problem, this is still an example application that hopes to inspire others into building more user friendly applications.   
+
+## Tavern smart contract
+
+Named after the classic location where gamers go and collect quests, Tavern is a smart contract that allows any Solidity developer to easily extend and make their own custom quest for their custom application. For more detailed information visit the repo for [Tavern](https://github.com/pokt-network/tavern).
+
+## Non-Fungible Tokens (NFT's)
+
+NFT's are based off of an [Ethereum Improvement Proposal](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md) standard popularized by [CryptoKitties](https://www.cryptokitties.co/). Otherwise known as ERC721, NFT's enable unique properties that normal EIP20 tokens do not offer, namely unique metadata that gives tokens within the same family vastly different values based on scarcity. Each BANANO token is an NFT, and depending on how difficult and rare the quest is, can potentially have vastly different value than a common BANANO Token. 
+
+NFT's introduce new game mechanics, such as burning or breeding the tokens for another type of NFT. For example, there are [dragons](https://hyperdragons.alfakingdom.com/) that you can feed your cryptokitties to for upgrades. One can imagine a game where someone creates an epic quest where you must collect a CrytoKitty, a BANANO and some other NFT to create a super NFT. The sky is the limit for this space and there is a significant amount of research being done.  
+
+## Pocket iOS SDK
+
+BANANO Quest uses Pocket's iOS SDK. 
+
+## Pocket Ethereum iOS plugin
+
```

Comment:
Don't we need more here? Here's the link: https://github.com/pokt-network/pocket-ios-eth